addon-test-support/helpers.js

```diff
@@ -189,10 +189,10 @@ export function find(selector, contextEl) {
   } else if (Object.prototype.toString.call(selector) === "[object String]") {
     selected = document.querySelectorAll(`${settings.rootElement} ${selector}`);
   }
-  if (!settings.useJQueryWrapper) {
-    return (selected.length === 1) ? selected[0] : selected;
+  if (settings.useJQueryWrapper) {
```

Comment:
Yeah, added support for a jQuery wrapped set via a setting was simply an experiment for the question: What if I could import these helpers in an existing acceptance test (perhaps in a large test suite) and it just works?

I do favor a native `find` helper that returns one element or a list of elements. However I'm curious if we could use these helpers for both integration testing and acceptance.

@rwjblue @cibernox should we consider the above as a use case for this library?